### PR TITLE
1824: /reviewers N should remove ready status for clean backports

### PR DESCRIPTION
**Pull request by zhaosongzs**
Currently, If a user issues /reviewers command in a clean backport, the reviewers requirement will not be displayed in the pr body and it won't be an integration blocker.

In this patch, after user issued /reviewers command, the reviewrs check for clean backport will be enabled. And if the Reviewers check requirements in the jcheck conf are stricter than the user's requirements, the bot will enforce the stricter requirements

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Issue
 * [SKARA-1824](https://bugs.openjdk.org/browse/SKARA-1824): /reviewers N should remove ready status for clean backports

### Reviewers
 * [Erik Joelsson](https://openjdk.org/census#erikj) (@erikj79 - **Reviewer**) ⚠️ Review applies to [aac73813](https://git.openjdk.org/skara/pull/1482/files/aac7381365bde0e861b3e3412668bc0d07e5222c)
 * [Guoxiong Li](https://openjdk.org/census#gli) (@lgxbslgx - Committer) ⚠️ Review applies to [aac73813](https://git.openjdk.org/skara/pull/1482/files/aac7381365bde0e861b3e3412668bc0d07e5222c)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/skara pull/1482/head:pull/1482` \
`$ git checkout pull/1482`

Update a local copy of the PR: \
`$ git checkout pull/1482` \
`$ git pull https://git.openjdk.org/skara pull/1482/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1482`

View PR using the GUI difftool: \
`$ git pr show -t 1482`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/skara/pull/1482.diff">https://git.openjdk.org/skara/pull/1482.diff</a>

</details>

### CODE REVIEW

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java [hunk 1]
```diff
@@ -141,6 +141,10 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
 
         reply.println(ReviewersTracker.setReviewersMarker(numReviewers, role));
         var totalRequired = formatLimits.values().stream().mapToInt(Integer::intValue).sum();
+        if (pr.labelNames().contains("clean") && pr.labelNames().contains("backport")) {
+            reply.println("Warning: The /reviewers command is used in a clean backport, it will enable reviewers check for this pr. " +
+                    "If the reviewers check requirements in the jcheck conf are stricter than your requirements, the bot will enforce the stricter requirements.");
```
Comment (erikj79) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java:146]:
```suggestion
            reply.println("Warning: By issuing the /reviewers command in this clean backport pull request, the reviewers check has now been enabled.");
```
I don't think we need to point out the application of stricter requirements in the reply. It's enough to document this on the wiki. I see I asked for it in the bug, but thinking on this more, the /reviewers command has never been able to reduce reviewer requirements from the minimum specified in .jcheck/conf.

**Review by erikj79 — COMMENTED** [review]
We will likely get a followup request to be able to revert `/reviewers X` in a clean backport. Implementing that seems less straight forward, so I'm happy to leave that for later.

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java (hunk 1 above)
Comment (zhaosongzs) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java:146]: > I don't think we need to point out the application of stricter requirements in the reply. It's enough to document this on the wiki. I see I asked for it in the bug, but thinking on this more, the /reviewers command has never been able to reduce reviewer requirements from the minimum specified in .jcheck/conf.

Yes, it's better. Thx!

**Review by lgxbslgx — APPROVED** [review]
Looks good.

**Review by erikj79 — APPROVED** [review]

**Pull request closed without merging**